snippets/controladdin.json

```diff
@@ -0,0 +1,31 @@
+{
+    "Snippet: Control Add-In": {
+        "prefix": "tcontroladdin",
+        "body": [
+            "controladdin ${1:MyControlAddIn}",
+            "{",
+            "\tRequestedHeight = ${3:300};",
+            "\tMinimumHeight = ${3:300};",
+            "\tMaximumHeight = ${3:300};",
+            "\tRequestedWidth = ${3:700};",
+            "\tMinimumWidth = ${3:700};",
+            "\tMaximumWidth = ${3:700};",
+            "\tVerticalStretch = ${4:true};",
+            "\tVerticalShrink = ${4:true};",
+            "\tHorizontalStretch = ${4:true};",
+            "\tHorizontalShrink = ${4:true};",
+            "\tScripts = ",
+            "\t\t'${5:script1.js}',",
+            "\t\t'${6:script2.js}';",
+            "\tStyleSheets =",
+            "\t\t'${7:style1.css}';",
+            "\tStartupScript = '${5:script1.js}';",
+            "\tRecreateScript = '${5:script1.js}';",
+            "\tRefreshScript = '${5:script1.js}';",
+            "\tevent ${8:MyEvent()};",
+            "\tprocedure ${9:MyProcedure()};",
+            "}"
+        ],
+        "description": "Snippet to create Control Add-In object"
```

Comment:
In the other snippets, we only have "Snippet: {Snippet name}" in the description. Could you change it to "Snippet: Control Add-In" to make it consistent?